internal/pkg/cli/env_upgrade.go

```diff
@@ -4,530 +4,28 @@
-}
-
-func (o *envUpgradeOpts) isDefaultLegacyTemplate(cfn envTemplater, appName, envName string) (bool, error) {
-	defaultLegacyEnvTemplate, err := o.legacyEnvTemplater.Template()
-	if err != nil {
-		return false, fmt.Errorf("generate default legacy environment template: %v", err)
-	}
-	actualTemplate, err := cfn.EnvironmentTemplate(appName, envName)
-	if err != nil {
-		return false, fmt.Errorf("get environment %s template body: %v", envName, err)
-	}
-	return defaultLegacyEnvTemplate == actualTemplate, nil
-}
-
-func (o *envUpgradeOpts) listLBWebServices() ([]string, error) {
-	services, err := o.store.ListServices(o.appName)
-	if err != nil {
-		return nil, fmt.Errorf("list services in application %s: %v", o.appName, err)
-	}
-	var lbWebServiceNames []string
-	for _, svc := range services {
-		if svc.Type != manifest.LoadBalancedWebServiceType {
-			continue
-		}
-		lbWebServiceNames = append(lbWebServiceNames, svc.Name)
-	}
-	return lbWebServiceNames, nil
-}
-
-func (o *envUpgradeOpts) upgradeLegacyEnvironmentWithVPCOverrides(upgrader legacyEnvUpgrader, conf *config.Environment,
-	fromVersion, toVersion string, albWorkloads []string) error {
-	if conf.CustomConfig != nil {
-		if err := upgrader.UpgradeLegacyEnvironment(&deploy.CreateEnvironmentInput{
-			Version: toVersion,
-			App: deploy.AppInformation{
-				Name: conf.App,
-			},
-			Name:              conf.Name,
-			ImportVPCConfig:   conf.CustomConfig.ImportVPC,
-			AdjustVPCConfig:   conf.CustomConfig.VPCConfig,
-			ImportCertARNs:    conf.CustomConfig.ImportCertARNs,
-			AllowVPCIngress:   conf.CustomConfig.EnableInternalALBVPCIngress,
-			CFNServiceRoleARN: conf.ExecutionRoleARN,
-		}, albWorkloads...); err != nil {
-			return fmt.Errorf("upgrade environment %s from version %s to version %s: %v", conf.Name, fromVersion, toVersion, err)
-		}
-		return nil
-	}
-	// Prior to #1433, we did not store the custom VPC config in SSM.
-	// In this situation we unfortunately have to ask the users to enter the VPC configuration into SSM or re-create the
-	// environment in case they run into this issue.
-	log.Warningln(`
-Looks like you've an environment with a customized VPC configuration.
-Copilot could not upgrade your environment's CloudFormation template.
-To learn more about how to fix it: https://github.com/aws/copilot-cli/issues/1601`)
-	return errors.New("cannot upgrade environment due to missing vpc configuration")
-}
-
+// Deprecated. Clients should use "env deploy" instead after copilot v1.20.0.
 // buildEnvUpgradeCmd builds the command to update environment(s) to the latest version of
 // the environment template.
 func buildEnvUpgradeCmd() *cobra.Command {
-	vars := envUpgradeVars{}
+	var envName, appName string
+	var all bool
 	cmd := &cobra.Command{
-		Use:    "upgrade",
-		Short:  "Upgrades the template of an environment to the latest version.",
+		Use:   "upgrade",
+		Short: "Upgrades the template of an environment to the latest version.",
```

Comment:
should mention deprecated here as well? i don't think the short description will ever print anyways but to be safe